### ISSUE TITLE
windows.h or unistd.h

### ISSUE THREAD

**Issue by aelzenaar** [HappyZombies/QuestGame#6]
Currently in basic.cpp we use both SetConsoleTitle (from windows.h) and chdir (from unistd.h). These two include files are from different OSes, so currently the code won't compile on either!

There is no standard way to set console title on Linux, and  the documentation for chdir in windows is [here](https://msdn.microsoft.com/en-us/library/windows/desktop/aa363806%28v=vs.85%29.aspx).

**Comment by gigaraptor** [comment]
well it compiles on my os, Windows 8.1. Do you use VS or MinGW?

**Comment by HappyZombies** [comment]
It compiles for me, Windows 8.1

**Comment by aelzenaar** [comment]
MinGW includes a generic GNU include stack with unistd. It prob won't work with VS, although I'm not sure. It definitely will *not* compile on Linux without a significant support stack (there are windows emulation libraries, but they add huge amounts of overhead for small (i.e. non-enterprise) projects).

**Comment by gigaraptor** [comment]
Yeah, we were just stating it was possible(at least I was). I think remove Windows.h as it is non POSIX compliant iirc. Unistd works on a variety of OSes while Windows obviously doesn't. 

As for VS, this is stallmanyouth we are talking about here, not Microsoft minors.

**Comment by aelzenaar** [comment]
What we could do is pull the OS-specific stuff into a header and then use #defines to include the right one depending on the compiler...

**Comment by gigaraptor** [comment]
While we could do that, I think that could end up over complicating the code. Most of the contributors on this have been programming for less than a year and the last thing we want is to confuse them with preprocessor directives they don't neccessarily even know.
